src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -193,9 +193,8 @@ internal T WithHttpManager(IHttpManager httpManager)
         /// https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
         /// <param name="options">Options for the internal MSAL token caches. </param>
-#if !SUPPORTS_CUSTOM_CACHE || WINDOWS_APP
+        [Obsolete("Use ITokenCache.SetCacheOptions(CacheOptions) method instead", false)]
```

Comment:
@jmprieur for obsolete message

Reply:
I think we need an aka.ms link here, @bgavrilMS. I had to think 1 min on how I would act on this message